src/datapane/common/utils.py

```diff
@@ -12,15 +12,15 @@
 import typing as t
 from contextlib import contextmanager
 from pathlib import Path
-from tempfile import NamedTemporaryFile, TemporaryDirectory, mkstemp
+from tempfile import NamedTemporaryFile, TemporaryDirectory, _TemporaryFileWrapper, mkstemp
 
 import chardet
 import importlib_resources as ir
 from chardet.universaldetector import UniversalDetector
 
 from .dp_types import MIME, DPMode, NPath, SDict, get_dp_mode
 
-mimetypes.init(files=[ir.files("datapane.resources") / "mime.types"])
+mimetypes.init(files=[str(ir.files("datapane.resources") / "mime.types")])
```

Comment:
I'm not sure why mypy is complaining about these Path / str issues in the standard library - they are all correct (the code would crash otherwise). Is it an issue with supporting Python 3.7 or something?

Reply:
Yep got it - that's fine, let's try to cast these all as `Path` objects if possible, we don't want to use `str` to represent paths as much as possible